pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1610,3 +1633,15 @@ func admitMigrationUpdate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRespons
 func ServeMigrationUpdate(resp http.ResponseWriter, req *http.Request) {
 	serve(resp, req, admitMigrationUpdate)
 }
+
+func ValidateDuplicateDHCPPrivateOptions(PrivateOptions []v1.DHCPPrivateOptions) error {
+	isUnique := map[int]bool{}
+	for _, index := range PrivateOptions {
+		if isUnique[index.Option] == true {
+			return fmt.Errorf("You have provided duplicate DHCPPrivateOptions")
+		} else {
```

Comment:
I think you can remove the else from here no?

Reply:
yep, else isn't required! Removed it :)

Reply:
changed index to DHCPPrivateOption here as well.